integrationTests/vm/iele/vmAgar_test.go

```diff
@@ -9,11 +9,12 @@ import (
 	"testing"
 
 	"github.com/ElrondNetwork/elrond-go/integrationTests/vm"
+	"github.com/ElrondNetwork/elrond-go/logger"
 	"github.com/ElrondNetwork/elrond-go/process/factory"
 	"github.com/stretchr/testify/assert"
 )
 
-var agarioFile = "../../agarioV3.hex"
+var agarioFile = "../../agar_v1_min.hex"
```

Comment:
So we had bugs in agarioV3?

Reply:
No. we have error in the delete process from trie. agarioV3 calls this multiple times.